web/rainmaker/dev-packages/egov-tradelicence-dev/src/ui-config/screens/specs/utils/index.js

```diff
@@ -1715,78 +1717,25 @@ export const updateDropDowns = async (
       console.log(e);
     }
   }
-
-  const tradeSubTypes = get(
-    payload,
-    "Licenses[0].tradeLicenseDetail.tradeUnits",
-    []
-  );
-
-  if (tradeSubTypes.length > 0) {
-    try {
-      tradeSubTypes.forEach((tradeSubType, i) => {
-        const tradeCat = tradeSubType.tradeType.split(".")[0];
-        const tradeType = tradeSubType.tradeType.split(".")[1];
-        set(payload, `LicensesTemp.tradeUnits[${i}].tradeType`, tradeCat);
-        set(payload, `LicensesTemp.tradeUnits[${i}].tradeSubType`, tradeType);
-
-        dispatch(
-          prepareFinalObject(
-            "applyScreenMdmsData.TradeLicense.TradeCategoryTransformed",
-            objectToDropdown(
-              get(
-                state.screenConfiguration.preparedFinalObject,
-                `applyScreenMdmsData.TradeLicense.filteredTradeTypeTree.${tradeCat}`,
-                []
-              )
-            )
-          )
-        );
-
-        dispatch(
-          prepareFinalObject(
-            "applyScreenMdmsData.TradeLicense.TradeSubCategoryTransformed",
-            get(
-              state.screenConfiguration.preparedFinalObject,
-              `applyScreenMdmsData.TradeLicense.filteredTradeTypeTree.${tradeCat}.${tradeType}`,
-              []
-            )
-          )
-        );
-        payload &&
-          dispatch(
-            prepareFinalObject(
-              `LicensesTemp.tradeUnits[${i}].tradeType`,
-              tradeCat
-            )
-          );
-
-        payload &&
-          dispatch(
-            prepareFinalObject(
-              `LicensesTemp.tradeUnits[${i}].tradeSubType`,
-              tradeType
-            )
-          );
-      });
-    } catch (e) {
-      console.log(e);
-    }
-  }
   setOwnerShipDropDownFieldChange(state, dispatch, payload);
 };
 
 export const getDocList = (state, dispatch) => {
```

Comment:
This function is independant of the changes you are working on. Please recheck whats the dependency here.